lxd/seccomp.go

```diff
@@ -344,7 +351,8 @@ delete_module errno 38
 const SECCOMP_NOTIFY_POLICY = `mknod notify [1,8192,SCMP_CMP_MASKED_EQ,61440]
 mknod notify [1,24576,SCMP_CMP_MASKED_EQ,61440]
 mknodat notify [2,8192,SCMP_CMP_MASKED_EQ,61440]
-mknodat notify [2,24576,SCMP_CMP_MASKED_EQ,61440]`
+mknodat notify [2,24576,SCMP_CMP_MASKED_EQ,61440]
+setxattr notify [3,1,SCMP_CMP_EQ]`
```

Comment:
We should put a line break at the end of that last line, otherwise the generate seccomp policy misses the final line break and reads funny with cat and others. (I know it's not a new problem, but since you're changing it, might as well fix it :)).

Reply:
Fixed.